fe1-web/src/features/rollCall/screens/RollCallOpened.tsx

```diff
@@ -71,13 +71,18 @@ const RollCallOpened = () => {
     addOwnToken().catch((e) => console.error(e));
   }, [lao, rollCallID, toast]);
 
-  const handleError = (err: string) => {
+  const handleError = (err: any) => {
     console.error(err);
-    toast.show(err, {
-      type: 'danger',
-      placement: 'top',
-      duration: FOUR_SECONDS,
-    });
+    // The "err" object might be an exception, take the message property if it exists
+    toast.show(
+      err?.message ||
+        (typeof err === 'string' ? err : 'Unkown error, please check the console and report it!'),
```

Comment:
are you sure you need all of this logic ?
I would tend to believe that `err.toString()` is probably enough to achieve this ?

Reply:
Now I also tend to believe that !